datahub-web-react/src/app/entity/shared/tabs/Lineage/generateUseSearchResultsViaRelationshipHook.ts

```diff
@@ -41,6 +45,7 @@ export default function generateUseSearchResultsViaRelationshipHook({
             variables: {
                 input: inputFields,
             },
+            skip: !filtersExist(filters, orFilters), // If you don't include any filters, we shound't return anything :). Might as well skip!
```

Comment:
only concern is that if we ever remove default filters from this tab (ie. we direct you to have 1 degree or whatever first right now) then we won't fire any query

Reply:
yep - agree

Reply:
problem right now is worse - we fetch searchAcrosslineage (our most expensive query) 2 times!